src/components/iframe-layout.vue

```diff
@@ -6,6 +6,10 @@
       la plateforme 1jeune1solution
     </a>
   </div>
+  <div id="iframe-footer">
+    <p>Accédez à nos&nbsp;</p
+    ><a href="https://mes-aides.1jeune1solution.beta.gouv.fr/cgu">CGU</a>
+  </div>
```

Comment:
```suggestion
    </a>
     <p>Accédez à nos <a href="https://mes-aides.1jeune1solution.beta.gouv.fr/cgu">CGU</a></p>
  </div>
```

En ajoutant éventuellement un `flex-direction: column;` au besoin pour que les deux liens restent l'un en dessous de l'autre